statsmodels/genmod/generalized_linear_model.py

```diff
@@ -607,9 +607,14 @@ def hessian(self, params, scale=None, observed=None):
             else:
                 observed = True
 
+        if getattr(self, '_tmp_like_exog', None) is not None:
+            tmp = self._tmp_like_exog
+        else:
+            tmp = np.empty_like(self.exog)
```

Comment:
you can make this part into a one-liner
`tmp = getattr(self, '_tmp_like_exog', np.empty_like(self.exog))

Reply:
I should have noticed that... Clever.